.github/workflows/build-package.yml

```diff
@@ -89,7 +89,7 @@ jobs:
         platform: ["arm64","amd64"]
         # These are our target environments
         # FIXME: renovate rule for this would be awesome
-        chroot: ["fedora-41", "fedora-42", "epel-10"]
+        chroot: ["fedora-42", "fedora-43", "epel-10"]
```

Comment:
Do we still need 41 for GTS?

Reply:
41 has made it's last release, next gts is 42 on tuesday so we're good.